packages/jest-each/src/__tests__/array.test.ts

```diff
@@ -32,14 +32,18 @@ const getGlobalTestMocks = () => {
   globals.it.skip = jest.fn();
   globals.describe.only = jest.fn();
   globals.describe.skip = jest.fn();
+  globals.test.concurrent = jest.fn();
+  globals.it.concurrent = jest.fn();
   return globals;
 };
 
 describe('jest-each', () => {
   [
     ['test'],
+    ['test', 'concurrent'],
```

Comment:
Probably want to add test cases for

`test.concurrent.skip` and `test.concurrent.only`

Reply:
An example on how to test `skip` would be great

Reply:
@Mark1626 these comments are addressed, right? The tests on line 45 and 46